### PR TITLE
C#: Change asp.net core view generation to be opt out

### PR DESCRIPTION
**Pull request by tamasvajk**

### CODE REVIEW

File: csharp/ql/integration-tests/all-platforms/cshtml_standalone_net6/cshtml.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```
Comment (hvitved) [inline csharp/ql/integration-tests/all-platforms/cshtml_standalone_net6/cshtml.csproj:5]: Should this say `net6.0`?

**Review by hvitved — COMMENTED** [review]
One question, otherwise LGTM.

File: csharp/ql/integration-tests/all-platforms/cshtml_standalone_net6/cshtml.csproj (hunk 1 above)
Comment (tamasvajk) [reply csharp/ql/integration-tests/all-platforms/cshtml_standalone_net6/cshtml.csproj]: Thanks. Yes that makes sense. The .net 6 sdk has no knowledge of `net8.0` at all, so it has an impact on the success of the initial `dotnet restore`.

I pushed another change too to merge some asp.net core detection checks.

**Review by michaelnebel — APPROVED** [review]
LGTM!

**Review by hvitved — APPROVED** [review]

**Pull request merged**